tests/unit/kvmagent/test_mevoco_handlers.py

```
@@ -0,0 +1,1352 @@
+    body = json.dumps(body_dict or {})
+    return {http.REQUEST_BODY: body, http.REQUEST_HEADER: {}}
+
+
+def _make_plugin() -> _MevocoPluginProto:
+    lock_mod = cast(_LockModule, cast(object, importlib.import_module("zstacklib.utils.lock")))
+    plugin_mod = cast(object, importlib.import_module("zstacklib.utils.plugin"))
+
+    from tests.conftest import passthrough_lock
+
+    lock_mod.lock = passthrough_lock
+    setattr(plugin_mod, "completetask", passthrough_lock)
```

Ruff (0.15.2)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)